# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../ash/denoise.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Remove some noise. With some C libraries the str* and bzero colide.

  - Rene Rebe <rene@exactcode.de>

diff -ur ash-linux-0.2/error.c ash-linux-0.2-fixed/error.c
--- ash-linux-0.2/error.c	1993-09-06 11:04:22.000000000 +0200
+++ ash-linux-0.2-fixed/error.c	2005-02-02 11:03:31.629060136 +0100
@@ -100,7 +100,7 @@
 		return;
 	}
 	intpending = 0;
-#ifdef BSD
+#ifdef BSD_NONO
 	sigsetmask(0);
 #endif
 	if (rootshell && iflag)
diff -ur ash-linux-0.2/mystring.h ash-linux-0.2-fixed/mystring.h
--- ash-linux-0.2/mystring.h	1993-08-24 11:12:53.000000000 +0200
+++ ash-linux-0.2-fixed/mystring.h	2005-02-02 11:01:35.411727856 +0100
@@ -44,14 +44,9 @@
 #ifdef __STDC__
 void scopyn(const char *, char *, int);
 char *strchr(const char *, int);
-void mybcopy(const pointer, pointer, int);
 int prefix(const char *, const char *);
 int number(const char *);
 int is_number(const char *);
-int strcmp(const char *, const char *);	/* from C library */
-char *strcpy(char *, const char *);	/* from C library */
-int strlen(const char *);		/* from C library */
-char *strcat(char *, const char *);	/* from C library */
 #else
 void scopyn();
 char *strchr();
@@ -67,4 +62,3 @@
 
 #define equal(s1, s2)	(strcmp(s1, s2) == 0)
 #define scopy(s1, s2)	((void)strcpy(s2, s1))
-#define bcopy(src, dst, n)	mybcopy((pointer)(src), (pointer)(dst), n)
